docs/zh/UserGuide/Operate-Metadata/Timeseries.md

```diff
@@ -304,3 +304,35 @@ It costs 0.004s
 ```
 
 > 注意，现在我们只支持一个查询条件，要么是等值条件查询，要么是包含条件查询。当然 where 子句中涉及的必须是标签值，而不能是属性值。
+
+创建对齐时间序列
+
+```
+create aligned timeseries root.sg1.d1(s1 INT32 tags(tag1=v1, tag2=v2) attributes(attr1=v1, attr2=v2), s2 DOUBLE tags(tag3=v3, tag4=v4) attributes(attr3=v3, attr4=v4))
```

Comment:
Hi, you can modify this part of the doc.
https://github.com/apache/iotdb/blob/2f784502e52e15548babb2e4b9133f0a0731ba44/docs/zh/UserGuide/Operate-Metadata/Timeseries.md?plain=1#L56-L66

Reply:
ok